packages/core/src/utils/markers.ts

```diff
@@ -0,0 +1,108 @@
+import Style from 'ol/style/Style'
+import Icon from 'ol/style/Icon'
+
+export interface MarkerStyle {
+  strokeWidth?: string | number
+  stroke?: string
+  fill?: string
+}
+
+export type GetMarkerFunction = (style: MarkerStyle, multi: boolean) => Style
```

Comment:
```suggestion
type GetMarkerFunction = (style: MarkerStyle, multi: boolean) => Style
```
Only used locally. If it should be used in any other place, please place it in the `types.ts`